packages/react-core/src/components/EmptyState/EmptyStateHeader.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/EmptyState/empty-state';
+import { EmptyStateIcon } from './EmptyStateIcon';
+
+export interface EmptyStateHeaderProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the EmptyStateHeader */
+  children?: React.ReactNode;
+  /** Additional classes added to the EmptyStateHeader */
+  className?: string;
+  /** Additional classes added to the title inside EmptyStateHeader */
+  titleClassName?: string;
+  /** Additional classes added to the Icon inside EmptyStateHeader */
+  iconClassName?: string;
```

Comment:
We should also add in an `iconColor` prop. Right now I don't believe there's anyway in the examples to change the icon color.

Reply:
We do have an EmptyStateIcon component with a color prop. I'd imagine that's the intended way we'd update icon color?

Reply:
Is this extending the react-icons size and color props? https://github.com/patternfly/patternfly-react/blob/781a8e7aceb126f9ef7cf8af23747169084f9748/packages/react-core/src/components/EmptyState/EmptyStateIcon.tsx#L6-L9

Assuming so, I believe @jenny-s51  is removing those props in https://github.com/patternfly/patternfly-react/pull/5275#pullrequestreview-1281925750

IMO any custom size or color or whatever that is outside of what the component supports with it's existing styles/variations should be handled some other way. The `<Icon>` component is probably the preferred way to handle an icon color/size change.

Reply:
@mcoker would it make more sense to just remove the `iconClassname` and `iconColor` props then, and just use the `icon` prop to pass whatever icon with the desired classes/color already set on it?

Reply:
I have removed the iconColor and iconClassName props and now we are passing the whole `<EmptyStateIcon icon={SomeIconComponent} />` as a ReactElement (with props set on the icon), instead of passing just `SomeIconComponent`